cypress/steps/yasqe-steps.ts

```diff
@@ -27,8 +27,8 @@ export class YasqeSteps {
         return cy.get('.yasqe_queryButton');
     }
 
-    static executeQuery() {
-        this.getExecuteQueryButton().click();
+    static executeQuery(index = 0) {
+        this.getExecuteQueryButton().eq(index).click();
```

Comment:
Can the "executeQueryButton" be more than one.

Reply:
Yes, it can. These buttons are bound to yasqe instance, so as many tabs are opened as many buttons there are.